doc/installing-a-development-environment.md

```diff
@@ -1,5 +1,12 @@
 ## Installing
 
+The development environment setup requires installation of the following software:
+
+1. Ruby - Needed by Jekyll. Jekyll is the static site generator that generates the website and docs.
+1. Node.js - Needs to be installed in order to install all the required development and javascript dependencies.
+1. Make - Can be installed in order to build cordova website using Makefile. This installation is optional.
+1. Python - Can be installed in order to invoke simple HTTP server from Makefile. This installation is optional.
```

Comment:
Grammar nitpick: please capitalise proper names for consistency. So:
- "cordova" -> "Cordova"
- "javascript" -> "JavaScript"
- etc.

Also: "using Makefile" should either be "using Make", or "using the Makefile".